.ci/test.sh

```diff
@@ -51,11 +51,20 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "check-docs" ]]; then
 fi
 
 if [[ $TASK == "lint" ]]; then
-    conda install -q -y -n $CONDA_ENV pycodestyle pydocstyle
+    conda install -q -y -n $CONDA_ENV \
+        pycodestyle \
+        pydocstyle \
+        r-argparse \
+        r-lintr
     pip install --user cpplint
+    echo "linting Python code"
```

Comment:
Capitalize messages (all three in this file)?

Reply:
> Also, I guess, that `.Rd` files should be re-generated.

Just regenerated the files. Note that to get past the r-lib/roxygen2#822, I had to install the latest `roxygen2`. Steps to build the docs on Mac (just documenting in case anyone is curious).

```
export CXX=/usr/local/bin/g++-8 CC=/usr/local/bin/gcc-8
Rscript build_r.R

# install roxygen2 from github
Rscript -e "devtools::install_github('r-lib/roxygen2')"
cd R-package
Rscript -e "roxygen2::roxygenise(load = 'installed')"
```